.orchestra/support/vscode-web/product.json

```diff
@@ -17,6 +17,7 @@
         "window.menuBarVisibility": "classic",
         "editor.foldingMaximumRegions": 65000,
         "files.autoSave": "off",
+        "keyboard.layout": "us",
```

Comment:
Mmmh, this doesn't sound like the best solution.

Is it possible at all to detect the layout correctly in the browser?
Have you looked into this in depth? This quite bad for UX.

Reply:
Dropped, made a proper fix on the vscode side

Reply:
What's the situation now on Chrome/Firefox?